### DISCUSSION TITLE
How Prevent GIF Images from Being Processed by markdownItEleventyImg Plugin

### DISCUSSION THREAD

**Discussion by joisun** [11ty/eleventy#3306] [Q&A]
Hello, Thanks for the awesome plugin.  

I am using Eleventy to write a blog and I have configured markdownItEleventyImg to handle images in my markdown files. My current configuration looks like this:

```js
const path = require('path');

markdownLib.use(markdownItEleventyImg, 
    {
    imgOptions: {
        widths: [1200],
        urlPath: `/${SITE_PREFIX}/images/`,
        outputDir: "./_site/images/",
        formats: ["jpeg", "gif"],
        sharpOptions: {
            animated: true,
        },
        // sharpJpegOptions:{
        //     quality:100
        // }
    },
    globalAttributes: {
        class: "markdown-image",
        decoding: "async",
        sizes: "100vw"
    },
    resolvePath: (filepath, env) => path.join(path.dirname(env.page.inputPath), filepath)
});
```

However, I have encountered an issue where all images, including GIFs, are processed and output in the specified formats. My intention is to exclude GIF images from being processed by this plugin, while still processing other image formats like JPEG.

I have attempted to adjust the configuration but haven't been successful in preventing GIFs from being processed.

Could you please provide guidance on how to configure markdownItEleventyImg to skip GIF images during processing (just generate the corret path)? 

> by the way, If my post have more then one gif asset, only the first one will be animated, which i have no idea too.

Thank you!

**Comment by zachleat** [comment]
Howdy! I think https://github.com/11ty/eleventy-img/issues/230 is a perfect spot for this discussion and we’ll it over there. Thanks!
